src/Site.js

```diff
@@ -940,13 +943,53 @@ Site.prototype.writeSiteData = function () {
   });
 };
 
+function constructDeploymentUrl(remoteRepo) {
+  const parts = remoteRepo.split('/');
+  let deploymentUrl = '';
+  if (parts[parts.length - 1].includes(GITHUB_IO_STRING)) {
+    // Use the github.io link itself
+    deploymentUrl = `https://${parts[parts.length - 1].substring(0, parts[parts.length - 1].length - 4)}`;
+  } else {
+    // Use the org_name + repo_name to reconstruct the url
+    deploymentUrl = `https://${parts[parts.length - 2]}.${GITHUB_IO_STRING}/`
+      + `${parts[parts.length - 1].substring(0, parts[parts.length - 1].length - 4)}`;
+  }
+  return deploymentUrl;
+}
+
+/*
+Returns a Promise with the URL of the remote of the repository
+*/
+function getUrl() {
+  return new Promise((resolve) => {
+    Git.Repository.open('.git').then(((repo) => {
```

Comment:
The deployment may not always be called at the root directory, so `.git` may not always exist. We can change `.git` -> `./`.

Reply:
> We can change `.git` -> `./`.

Addressed

> Perhaps you can modify `docs/userGuide/tipsAndTricks.md`

Addressed

> If there are alternative packages/methods that allow us to avoid this problem completely, that would be the best, otherwise adding the above should be sufficient.

There was another package that did the job - [`git-remote-origin-url`](https://www.npmjs.com/package/git-remote-origin-url), but it messed around with the gitconfig file which consequently failed several tests. That's why I went ahead with `nodegit`.